limbo/config.py

```diff
@@ -0,0 +1,50 @@
+import os
+from backports import configparser as confparser
+
+
+class Config:
+    env_config = {}
```

Comment:
This should be an instance variable, not a class-level variable. Later on, when you say `self.env_config`, you're not referring to this variable but creating a new instance variable; this one is available as `Config.env_config` but is never used